src/main/java/org/openmrs/isanteplus/performancedata/model/Obs.java

```diff
@@ -73,9 +72,9 @@
 
     private String voidReason;
 
-    private UUID uuid;
+    private String uuid;
```

Comment:
toString of UUID does not match mysql insert syntax. I can wrap it up in some custom class and override toString method, but String works just fine as well.